src/pymor/discretizers/builtin/cg.py

```diff
@@ -995,6 +995,11 @@ def discretize_stationary_cg(analytical_problem, diameter=None, domain_discretiz
         Must be provided if `grid` is specified.
     preassemble
         If `True`, preassemble all operators in the resulting |Model|.
+    energy_product
```

Comment:
I am not sure if this is such good name. After all, it is not an 'energy_product' that should be passed. So maybe 'mu_energy_product'?